Cargo.toml

```diff
@@ -11,8 +11,8 @@ async-channel = "2.1"
 async-trait = "0.1"
 bincode = "1"
 chrono = { version = "0.4", features = ["serde"] }
+clap = { version = "4.5", features = ["derive"] }
```

Comment:
I think it would be better to specify only the major version unless we have a certain reason to use version 4.5. It is mentioned in one of the Notion documents.

Reply:
Fixed as your comment.

@sophie-cluml I copied it from the Feature Sensor repository and it might need the adjustment as well.